### ISSUE TITLE
Add check for functional indexes lacking ANALYZE

### ISSUE THREAD

**Issue by rjuju** [OPMDG/check_pgactivity#269]
Following this thread: https://www.postgresql.org/message-id/flat/CAFcNs%2BqpFPmiHd1oTXvcPdvAHicJDA9qBUSujgAhUMJyUMb%2BSA%40mail.gmail.com

I'm wondering if we should add a check for that.

**Comment by ioguix** [comment]
Good idea. This could be spread to other kind of relation that can haz stats btw.

**Comment by rjuju** [comment]
Won't those be covered by check_last_maintenance?

**Comment by rjuju** [comment]
Maybe we should just add functional indexes to the scope of check_last_analyze.
